### ISSUE TITLE
HTML doc typos in Modelica.Mechanics.MultiBody

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#675]
**Reported by anonymous on 4 Jan 2012 10:08 UTC**
MSL 3.x

There is a HTML documentation typo in
```
Modelica.Mechanics.MultiBody.Joints.UniversalSpherical
Modelica.Mechanics.MultiBody.Joints.Assemblies.JointUPS
```

```
In practice this rotation is usually of no interested
```
->
```
In practice this rotation is usually of no interest
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/675

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 10 Feb 2012 12:30 UTC**

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 10 Feb 2012 12:30 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 10 Feb 2012 15:15 UTC**
Fixed as part of 5c6a97e8faf89b7acfb89d238064078a8eee423f.
